src/Compilers/Core/Portable/CommandLine/AnalyzerConfig.cs

```diff
@@ -222,6 +222,12 @@ internal sealed class Section
             /// </summary>
             public static StringComparison NameComparer { get; } = StringComparison.Ordinal;
 
+            /// <summary>
```

Comment:
I can't put a comment on the real location, but above there's a property `IsRoot` for checking if the file is a root editorconfig. Why not an `IsGlobal` property for the same?

Reply:
Just felt like it was mixing up responsibilities in the original design, it's probably fine, but it still feels a bit weird to me. We say AnalyzerConfig is this fully formed thing, but then it's got responsibility for telling the combiner if it should be combined or not. 

---
In reply to: [423390636](https://github.com/dotnet/roslyn/pull/43889#discussion_r423390636) [](ancestors = 423390636)

Reply:
But whether or not something's a global editorconfig has nothing to do with whether or not it's combined. How do I know if I should universally apply this editorconfig? Or if the section names should be considered globs or absolute paths? is_global is a property of a particular editorconfig, just like is_root

Reply:
Another way of looking at it: can I answer this question just looking at a single editorconfig? If so, probably belongs on AnalyzerConfig. If I need to see the complete set of editorconfigs, probably belongs on AnalyzerConfigSet. So if you want to preprocess the global editorconfigs, that belongs in AnalyzeConfigSet, but if you want to know if a single editorconfig is global, that belongs on AnalyzerConfig